lib/fluent/plugin/out_null.rb

```diff
@@ -18,10 +18,48 @@
 
 module Fluent::Plugin
   class NullOutput < Output
+    # This plugin is for tests of non-buffered/buffered plugins
     Fluent::Plugin.register_output('null', self)
+    Fluent::Plugin.register_output('buffered_null', self)
```

Comment:
Why `buffered_null` is needed?
This is introduced since v0.14 and there is no production use.
So we don't need to keep the compatibility.